### PR TITLE
New package: Databricks.Databricks version 0.212.0

### LINKED ISSUE #135161: [Package Request]: Databricks CLI
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/databricks/cli/releases/tag/v0.212.0
Publisher: Databricks
Package Name: DatabricksCLI
Description: Databricks Command Line Interface
Package Version: 0.212.0
Installer URL: https://github.com/databricks/cli/releases/download/v0.212.0/databricks_cli_0.212.0_windows_amd64.zip
Checksums: https://github.com/databricks/cli/releases/download/v0.212.0/databricks_cli_0.212.0_SHA256SUMS

I have a couple of questions:
- if the app is a portable `*.exe`, that just has to be placed in a $PATH, do we need any installer to wrap it?
- is the code signing required for the package to be included on `winget`?
- what could be done to automate `winget` to pick up new versions of package as soon as they are released? can this repo just poll the github releases page somehow?

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:

- Resolve: #135161
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135163)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Review by nfx — CHANGES_REQUESTED** [review]
Could we rename the package to `Databricks.DatabricksCLI`?

**Comment by stephengillie** [comment]
This looks like a valid PR to me. I don't see a cause for these uncommon labels.

File: manifests/d/Databricks/DatabricksCLI/0.212.0/Databricks.Databricks.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: databricks.exe
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2024-01-11
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/databricks/cli/releases/download/v0.212.0/databricks_cli_0.212.0_windows_amd64.zip
+  InstallerSha256: 362D731C8C1D3DBA583B5305A6159771E1BF450CB0C18D31B7E56FA919C5D479
```
Comment (nfx) [inline manifests/d/Databricks/DatabricksCLI/0.212.0/Databricks.Databricks.installer.yaml:16]: hash matches

File: manifests/d/Databricks/DatabricksCLI/0.212.0/Databricks.Databricks.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Architecture: arm64
+  InstallerUrl: https://github.com/databricks/cli/releases/download/v0.212.0/databricks_cli_0.212.0_windows_arm64.zip
+  InstallerSha256: D24EEB5B415A1C7F96253AC3C6BB58B95C76688962CD03DDA53C13BCEEAEEE88
```
Comment (nfx) [inline manifests/d/Databricks/DatabricksCLI/0.212.0/Databricks.Databricks.installer.yaml:19]: hash matches

**Review by nfx — APPROVED** [review]
lgtm

**Review by nfx — APPROVED** [review]
lgtm 🚀

**Comment by nfx** [comment]
@stephengillie, please merge once you have a chance

**Comment by nfx** [comment]
@Exorcism0666 ^

**Comment by Exorcism0666** [comment]
@nfx It's normal, at the moment there are some problems managing pull requests and therefore this is buggy, so we have to wait some more.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**